draft-ietf-tls-tls13.md

```diff
@@ -796,7 +797,9 @@ of the digest output. Note that previous versions of TLS used RSASSA-PKCS1-v1_5,
 not RSASSA-PSS.
 
 All ECDSA computations MUST be performed according to ANSI X9.62 {{X962}}
-or its successors.  Data to be signed/verified is hashed, and the
+or its successors.  The utmost care must be taken to choose high-entropy
+"k" values; use of {{RFC6979}} Deterministic ECDSA is RECOMMENDED to ensure this.
```

Comment:
put the name before the ref ("Deterministic ECDSA {{RFC6979}}" instead of above)